src/components/AutocompleteSelectMenu/AutocompleteSelectMenu.tsx

```diff
@@ -123,7 +123,6 @@ const AutocompleteSelectMenu: React.FC<AutocompleteSelectMenuProps> = props => {
                   onBlur: () => {
                     closeMenu();
                     setMenuPath([]);
-                    setInputValue(displayValue);
```

Comment:
This serves a purpose here, we want to fallback to the selected value on blur if user clears the input.